### ISSUE TITLE
When selecting top communities channel, they all appear selected

### ISSUE THREAD

**Issue by alaibe** [status-im/status-desktop#4166]
# Bug Report

## Description

![image](https://user-images.githubusercontent.com/491074/143257452-c5c40a14-0dd1-4188-8daf-b99ceac73271.png)

Only Desktop channel should appear selected

## Steps to reproduce

Go to a community with multiple channel, select a random one, then select the top one of the list

#### Expected behavior

Only top channel should appear selected

#### Actual behavior

Most channel appear selected

### Additional Information

- Status desktop version: release/0.4.0
- Operating System: Linux

**Comment by jrainville** [comment]
This is too weird. I can't figure it out. Putting it back in backlog

**Comment by alaibe** [comment]
@jrainville can you at least reproduce? I think @PascalPrecht worked on something similar during our first dogfooding, maybe the error is similar?

**Comment by PascalPrecht** [comment]
I can reproduce this. And I've seen a similar bug in other places. For some reason what seems to happen here is that the highlighted items get a hover state. I've seen this in some modals in the application, where when the modal is opened, one of the modal list items had a hover effect activated even though it hasn't been touched (yet). This goes away immediately once you actually hover it.

I don't know what causes this, could be a bug in QML. Needs to be investigated.

The issue I was working on in the past was were chat items would just disappear, or rather their texts. And if I remember correctly, this had something to do with the font that was being used.

**Comment by jrainville** [comment]
> @jrainville can you at least reproduce? I think @PascalPrecht worked on something similar during our first dogfooding, maybe the error is similar?

@alaibe yeah I can reproduce it every time

**Comment by jrainville** [comment]
> For some reason what seems to happen here is that the highlighted items get a hover state

yea exactly. I found the problem like you said, being that when the active channel changes to the first one, all the other items, except the last one, get the `mouseInside` flag set to true. And like Pascal, I can't figure out why. I tried to find why for an afternoon, but failed, that's why I unassigned. The priority is pretty low and it's very hard/weird to fix
